### PR TITLE
[Doc] Add short paragraphs at the beginning of each section

### LINKED ISSUE #189: Make sure each chapter and section has a short explanation paragraph at the beginning

### PR DESCRIPTION
**Pull request by Huy-Ngo**
Resolve #189

### CODE REVIEW

**Review by 9a24f0 — APPROVED** [review]
LGTM! No idea about how dr. Quang's thought

File: docs/source/meth/uc-anal-design/security.rst [hunk 1]
```diff
@@ -1,15 +1,15 @@
 Security
 ========
 
+Some designs that address security concerns -- authentication, authorization,
+encryption, preventing attacks -- are described in this part.
```
Comment (McSinyx) [inline docs/source/meth/uc-anal-design/security.rst:5]:
```suggestion
Some designs that address security concerns---authentication, authorization,
encryption, preventing attacks---are described in this part.
```

Have some class.

**Comment by Huy-Ngo** [comment]
thanks

File: docs/source/meth/database/index.rst [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
 ===============
 
-To design the database, we first analyze which entities need to be represented.
-From there, we then derive tables for the database accordingly.
+This section is dedicated to designing the database for the system.
+Designing a non-relational database can be tricky due to its flexibility, as opposed to
+for traditional relational database.
+However, we followed a similar approach:
+we first analyze which entities need to be represented and
+from there, we derive tables for the database accordingly.
```
Comment (McSinyx) [inline docs/source/meth/database/index.rst:9]: In addition to this, there should be some minimal linkage on which part of the db serves which part of the objectives, I think.

**Comment by McSinyx** [comment]
LOL I'm late, lemme do it then.

**Pull request merged**